website/docs/upgrades/1.0_to_1.1/changes_to_default_composition_order.md

```diff
@@ -62,9 +62,16 @@ foo:
 </div>
 
 
-### Migration
-For the majority of applications, this will not cause issues. If your application requires the previous behavior, 
-you can achieve it by adding `_self_` as the first item in your Defaults List:
+
+## Migration
+### Primary config is a YAML file
+To ensure this change is not missed by people migrating from Hydra 1.0, Hydra 1.1 issues a warning if the Defaults List in the primary config is missing `_self_`, and there are config values in addition to the Defaults List.
+To make this warning disappear, you need to add `_self_` to the Defaults List of the primary config.
+
+The first step is to determine if you get the same config by comparing the output of `--cfg job` on both versions of Hydra.
```

Comment:
perhaps move this to the desciprtion under migration

```
## Migration
If your application uses `hydra.main`, the best way to verify updating Hydra versions does not change your configurations is to run `python my_app.py --cfg job` to get the configuration on both the new and old Hydra versions and compare them.
If your application uses `compose` API, please make sure you have comprehensive unit tests on the composed configuration.

### Primary config is a YAML file

```

Reply:
Take a look at the last update.